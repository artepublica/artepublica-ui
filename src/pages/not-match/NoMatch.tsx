import { useEffect } from 'react';

import { StackActions, useNavigation } from '@react-navigation/native';

function NoMatch(): JSX.Element {
    const navigation = useNavigation();

    useEffect(() => {
        navigation.dispatch(StackActions.replace('NotFound'));
    }, []);

    return <></>;
}

export default NoMatch;
