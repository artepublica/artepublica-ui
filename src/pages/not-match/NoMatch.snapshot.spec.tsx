import { create } from 'react-test-renderer';

import NoMatch from './NoMatch';

jest.mock('@react-navigation/native', () => {
    return {
        ...jest.requireActual('@react-navigation/native'),
        useNavigation: () => ({
            dispatch: jest.fn(),
        }),
    };
});

describe('<NoMatch />', () => {
    it('default snapshot', () => {
        const json = create(<NoMatch />).toJSON();

        expect(json).toMatchSnapshot();
    });
});
