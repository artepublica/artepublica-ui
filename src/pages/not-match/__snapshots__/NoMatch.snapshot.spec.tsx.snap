// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoMatch /> default snapshot 1`] = `null`;
